rapids-cmake/cpm/versions.json

```
@@ -6,9 +6,10 @@
       "git_tag": "v${version}"
     },
     "CCCL": {
-      "version": "2.2.0",
-      "git_url": "https://github.com/NVIDIA/cccl.git",
-      "git_tag": "v${version}",
+      "version": "2.5.0",
+      "git_shallow": false,
+      "git_url": "https://github.com/trxcllnt/cccl.git",
+      "git_tag": "fea/build-rapids",
```

We need to switch back to the upstream CCCL repo and update the tag before merging. It seems like we could adopt the latest commit on `main`, since [CCCL PR 1667](https://github.com/NVIDIA/cccl/pull/1667) [is blocked by](https://github.com/NVIDIA/cccl/pull/1667#issuecomment-2150787054) [CCCL PR 1779](https://github.com/NVIDIA/cccl/pull/1779). Then this should be mergeable. We'll update to the next tag once it's available.
```suggestion
      "git_url": "https://github.com/NVIDIA/cccl.git",
      "git_tag": "fde1cf79bde6744b6739636f502b5edfefe302e1",
```